### DISCUSSION TITLE
Poderes concedidos estão com problema

### DISCUSSION THREAD

**Discussion by VitZinc** [YuriAlessandro/gerador-ficha-tormenta20#298] [Problemas]
Ola, os poderes concedidos estão sendo considerados como poderes gerais, ja vi uns paladino de valkaria com poder de sszass, khaly e khalmyr.
E vi uns inventores com poderes repetidos que não podem ser repetidos.

**Comment by YuriAlessandro** [comment]
A gente subiu uma correção com várias melhorias na seleção de poderes hoje. Pode ser que esses problemas ai que você citou tenham sido corrigidos.

Se você perceber eles novamente, seria muito bom algumas imagens com os exemplos, que fica mais fácil da gente corrigir aqui.

**Comment by VitZinc** [comment]
![image](https://user-images.githubusercontent.com/90149166/132553455-5e051b57-c9c8-45a8-b644-068d5a7e125b.png)
![image](https://user-images.githubusercontent.com/90149166/132553511-ec005b2b-748a-45f5-828b-88c650e542af.png)
Poderes concedidos de Aharadak :
Percepção Temporal: Você pode gastar 3 PM para adicionar seu bônus de Sabedoria (mínimo +1, limitado por seu nível) a seus ataques, Defesa e testes de Reflexos até o fim da cena.
Afinidade com a Tormenta: Você recebe +10 em testes de resistência contra efeitos da Tormenta e de suas criaturas
Ambos sendo ratados como poderes gerais
Poderes de Megalok:
Urro Divino: Quando faz um ataque ou lança uma magia, você pode pagar 1 PM para somar seu modificador de Constituição (mínimo +1) à rolagem de dano desse ataque ou magia.
Poder de Thwor:
Fúria Divina: Você pode gastar 2 PM para invocar uma fúria selvagem, tornando-se temível em combate. Você recebe +2 em testes de ataque e rolagens de dano corpo a corpo, mas não pode executar nenhuma ação que exija paciência ou concentração (como usar a perícia Furtividade ou lançar magias). A Fúria Divina termina se, ao fim da rodada, você não tiver atacado nem sido alvo de um efeito hostil
Poder se Sszzaas:
Sangue Ofídico: Você recebe resistência a veneno 5 e a CD para resistir aos seus venenos aumenta em +2.

Entre outros né. Os poderes concedidos são apenas se vc segue as obrigações e restrições daquele deus, podendo apenas seguir um deus, e pelo jeito não pode sair pegando eles com poder geral então tem algo de errado no codico, provavel que todos estejam sendo considerados poderes gerais.

**Reply by YuriAlessandro** [reply]
Consegui achar aqui o problema e resolvi

**Reply by VitZinc** [reply]
grande programador

**Reply by VitZinc** [reply]
![image](https://user-images.githubusercontent.com/90149166/132967818-a07078ff-0873-4fec-84a5-c5d78edd2717.png)
 Mas então ...

**Reply by YuriAlessandro** [reply]
Tá certo, vamos continuar revisando aqui

Muito obrigado por continuar reportando esse problema

**Reply by YuriAlessandro** [reply]
Agora a gente percebeu que, quando era devoto, ele pegava poderes de qualquer divindade quando upava. Acho que corrigimos isso. Será que ainda tem algum problema?
